modules/indexing/src/test/java/org/apache/ignite/internal/processors/cache/persistence/db/LongDestroyDurableBackgroundTaskTest.java

```diff
@@ -323,7 +323,7 @@ private void validateIndexes(Ignite ignite) {
         log.info("Doing indexes validation.");
 
         VisorValidateIndexesTaskArg taskArg =
-            new VisorValidateIndexesTaskArg(Collections.singleton("SQL_PUBLIC_T"), nodeIds, 0, 1);
+            new VisorValidateIndexesTaskArg(Collections.singleton("SQL_PUBLIC_T"), nodeIds, 0, 1, false);
```

Comment:
Are you sure about changing current behavior of the existing tests that use index validation? I doubt that it is good.
Are we sure about changing default behavior of validate-indexes command at all?